### PR TITLE
Update action.markdown

### PR DESCRIPTION
**Pull request by martin3000**
In my browser, "{{ notification_service }}" resulted in an empty service call.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/automation/action.markdown [hunk 1]
```diff
@@ -33,7 +33,7 @@ automation 2:
     notification_service: notify.paulus_iphone
   action:
     # Actions are scripts so can also be a list of actions
-    - service: "{{ notification_service }}"
+    - service: notify.notify
```
Comment (frenck) [inline source/_docs/automation/action.markdown]: This change is incorrect. The variable is defined just above. This example is an example service to show the feature of using automation variables.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by martin3000** [comment]
This is not visible in the web page. There is only: service: ""
See screenshot.
image

**Comment by frenck** [comment]
Aah, that is the issue. OK.

In that case, the issue is that the example isn't encapsulated with `{% raw %}` & `{% endraw %}`, causing our website to interpret it as Liquid instead of it being part of the code block.

Please add the raw tags instead.

../Frenck

**Comment by martin3000** [comment]
I don't know what liquid is so I don't play with things I don't know.

**Comment by frenck** [comment]
Added the needed adjustment.

**Comment by frenck** [comment]
Now it shows:

image

**Review by frenck — APPROVED** [review]

**Pull request merged**